apps/files/src/fileFilters.js

```diff
@@ -0,0 +1,13 @@
+export const fileFilters = [{
+  name: 'Files',
```

Comment:
still no translation :see_no_evil: ;-)

Reply:
`vue-gettext` has no global available getter AFAIK; so i needed to wrap it into a real SingleFileComponent, thus get the ViewModel access via `this`
I don't want to pollute the TopBar, so I've created a new File.

If you like it, i could squash it.